catalog/ui/search-ui/search-endpoint/pom.xml

```diff
@@ -388,9 +388,29 @@
                         </Private-Package>
                         <Export-Package/>
                         <Import-Package>
+                            ddf.action,
+                            ddf.catalog,
```

Comment:
@stustison No reason that I know of that we can't make the changes as you suggested. When I started the ticket I was attempting to remove the `*` and `resolution:=optional` when really I could have left the `*` after making sure the dependencies were correct. I'll try that now.